clayui.com/gatsby/createPages.js

```diff
@@ -39,9 +40,9 @@ const getTabs = (permalink, pathGroup) => {
 	return [
 		{
 			href: permalink,
-			name: 'React Component',
+			name: 'Overview',
 		},
-		...tabs.map(({node: {fields: {slug}}}) => ({
+		...tabs.reverse().map(({node: {fields: {slug}}}) => ({
```

Comment:
@kresimir-coko you can use `.sort` with specific rules, it seems more secure but I don’t care about keeping `reverse`, I think the only problem with this is that we rely on the reverse order of the elements and the loading of the files and GraphQL.

Reply:
I'm not sure how I can properly debug this, as the only output I'm getting is in the server log and that's outputting GraphQL values.

When I do something like this:
```js
const sortTabs = (a, b) => {
	console.log('a: ', a);

	if (a.name > b.name) {
		return 1;
	}

	if (a.name > b.name) {
		return -1;
	}
};
```

This is the output I get:
```bash
a:  [Object: null prototype] {
  node:
   [Object: null prototype] {
     fields:
      [Object: null prototype] {
        layout: '',
        nightly: false,
        redirect: '',
        redirectFrom: '',
        slug: 'docs/components/modal/api.html',
        mainTabURL: 'docs/components/modal.html' },
     body: ...
   } 
}
```

Am I missing something when debugging inside `createPages.js`? @matuzalemsteles

Reply:
```js
	if (a.name > b.name) {
		return 1;
	}

	if (a.name > b.name) {
		return -1;
	}
```

Note the repeated `>` operators.

Should be one `>` and one `<`, and then a `return 0` for good measure at the bottom.

Reply:
True true, that was a typo while trying to get proper output, just a placeholder, but thanks for noticing!

Reply:
@kresimir-coko to avoid this, try to `sort` after `map`, it will be easier to deal with.